movemeant/movemeant/settings.py

```diff
@@ -15,19 +15,17 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get("SECRET_KEY")
+SECRET_KEY = os.environ.get("SECRET_KEY", '1234')
```

Comment:
whoops sorry, let me know if you still need the .env

this just expects a .env file at the root dir of the project. with all the variables

Reply:
I added a default value there, just in case there's no .env file available, eg in the local dev environment